src/components/sbb-menu/sbb-menu.scss

```diff
@@ -112,21 +108,32 @@
     --sbb-menu-max-width: #{px-to-rem-build(320)};
     --sbb-menu-min-width: #{px-to-rem-build(180)};
 
-    top: var(--sbb-menu-position-y);
-    left: var(--sbb-menu-position-x);
+    top: 0;
+    left: 0;
     right: unset;
     bottom: unset;
+    max-height: fit-content;
     border-radius: var(--sbb-menu-border-radius);
+
+    &[open] {
+      top: var(--sbb-menu-position-y);
+      left: var(--sbb-menu-position-x);
+      max-height: max(120px, var(--sbb-menu-max-height));
```

Comment:
I preferred the solution where 120px was in the var `-sbb-menu-min-height: #{px-to-rem-build(120)};`.
What is the reason you removed it?